### DISCUSSION TITLE
Mismatch

### DISCUSSION THREAD

**Discussion by deulizealand** [revenkroz/telegram-web-app-bot-example#4] [Q&A] [answered]
Example provided doesnt match with example in the bot

**Comment by revenkroz** [comment, marked as answer]
Hello!

Because the backend for the bot can be anything and in any language (Go, PHP, Rust, Python, ...) and we are only interested in the frontend (WebApp), I left only the code that can be used without the backend. It's also worth mentioning that I published this example in the first hours after the release of this feature for Telegram, so something could change since then.

**Reply by deulizealand** [reply]
i mean, the given example ( index.html ) doesnt match with example in the bot you given. 
for example : in the bot there are test alerts section, but in github there's not exist.

apologize for my bad english

**Reply by revenkroz** [reply]
It was updated not long ago. I will update my example shortly. But it will still remain without a backend - just the frontend.

**Reply by revenkroz** [reply]
@deulizealand I updated the example code. I added missing features and comments to the code to make it clear what to do and what to change (for example, to interact with the backend).

**Reply by Jobians** [reply]
Ok thanks

**Reply by hendra-jti** [reply]
> @deulizealand I updated the example code. I added missing features and comments to the code to make it clear what to do and what to change (for example, to interact with the backend).

thankyou for your time

**Reply by deulizealand** [reply]
i forgot that i reply it with my another account. thanks for your time

**Comment by Jobians** [comment]
I have uploaded the full code in his demo app here

https://github.com/Jobians/Telegram-WebApp-Example/blob/main/Sample.html

**Reply by hendra-jti** [reply]
thankyou bro
